apps/web/src/components/UserProfile/UserProfile.tsx

```diff
@@ -130,427 +43,168 @@ const UserProfile = ({
+              <TableOfContents.AnchorLink id={PAGE_SECTION_ID.WORK_LOCATION}>
+                {intl.formatMessage(navigationMessages.workLocation)}
+              </TableOfContents.AnchorLink>
+            </TableOfContents.ListItem>
+            <TableOfContents.ListItem>
+              <TableOfContents.AnchorLink id={PAGE_SECTION_ID.WORK_PREFERENCES}>
+                {intl.formatMessage(navigationMessages.workPreferences)}
+              </TableOfContents.AnchorLink>
+            </TableOfContents.ListItem>
+            <TableOfContents.ListItem>
+              <TableOfContents.AnchorLink
+                id={PAGE_SECTION_ID.CAREER_TIMELINE_AND_RECRUITMENT}
+              >
+                {intl.formatMessage(
+                  navigationMessages.careerTimelineAndRecruitment,
+                )}
+              </TableOfContents.AnchorLink>
+            </TableOfContents.ListItem>
+            <TableOfContents.ListItem>
+              <TableOfContents.AnchorLink id={PAGE_SECTION_ID.SKILL_SHOWCASE}>
+                {intl.formatMessage(navigationMessages.skillShowcase)}
+              </TableOfContents.AnchorLink>
             </TableOfContents.ListItem>
-          )} */}
           </TableOfContents.List>
         </TableOfContents.Navigation>
-      )}
-      <TableOfContents.Content>
-        {subTitle}
-        {showSection("about") && (
+        <TableOfContents.Content>
           <TableOfContents.Section id={PAGE_SECTION_ID.ABOUT}>
-            <HeadingWrapper show={!!sections.about?.editUrl}>
-              <div
-                data-h2-flex-item="base(1of1) p-tablet(fill)"
-                data-h2-text-align="base(center) p-tablet(left)"
-              >
-                <TableOfContents.Heading
-                  as={headingLevel}
-                  icon={UserIcon}
-                  data-h2-margin-top="base(0)"
-                >
-                  {intl.formatMessage(navigationMessages.aboutMe)}
-                </TableOfContents.Heading>
-              </div>
-              {sections.about?.editUrl && (
-                <EditUrlLink
-                  link={sections.about.editUrl}
-                  text={intl.formatMessage(
-                    {
-                      defaultMessage: "Edit {title}",
-                      id: "3R3jKp",
-                      description: "Link to edit object",
-                    },
-                    {
-                      title: intl.formatMessage(navigationMessages.aboutMe),
-                    },
-                  )}
-                />
-              )}
-            </HeadingWrapper>
-            {sections.about?.override ? (
-              sections.about.override
-            ) : (
-              <AboutSection user={user} editPath={sections.about?.editUrl} />
-            )}
+            <TableOfContents.Heading
+              as={headingLevel}
+              icon={UserIcon}
+              data-h2-margin-top="base(0)"
+              data-h2-margin-bottom="base(x1)"
```

Comment:
Sorry, this is my lack of understand of how hydrogen works but, this should be:

`data-h2-margin="base(0 0 x1 0)"` and all the others should be `data-h2-margin="base(x1.5 0 x1 0)"`

Reply:
[b2b17d3](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10765/commits/b2b17d38c849ade58dda12db2e0512c22522168a)